widget/widgetAssets/html/timIntegrationConfig.html

```diff
@@ -0,0 +1,103 @@
+<!-- Copyright start
+  MIT License
+  Copyright (c) 2024 Fortinet Inc
+  Copyright end -->
+<div class="row padding-left-lg padding-right-lg margin-right-0">
+    <div data-ng-if="processing" class="text-center status-spinner">
+      <cs-spinner></cs-spinner>
+    </div>
+    <div data-ng-if="currentConnectorStatus === 'installed'">
+      <div data-ng-class="{'col-md-12': fromAgent, 'col-md-6': !fromAgent}">
+        <div>
+          <div data-ng-show="connector.config_count >= 0">
+            <div class="connector-form">
+              <div class="row">
+                <div class="col-md-6">
+                  <label for="configuration-selection" class="control-label configuration-selection">Select Configuration</label>
+                  <select class="form-control" id="configuration-selection" data-ng-disabled="processing"
+                    name="configurationsSelection" data-ng-model="selected.configuration"
+                    data-ng-options="configuration as configuration.name for configuration in connector.configuration track by configuration.id"
+                    data-ng-change="timConfigurationChanged($index, selected.configuration)">
+                    <option value="" data-ng-hide="fromAgent && connector.configuration.length === 1">{{input.selectionPlaceholder}}</option>
+                  </select>
+                </div>
+                </div>
+                <div class="clearfix"></div>
+              </div>
+              <div class="row margin-top-md">
+                <div class="col-md-5">
+                  <div class="connector-health">
+                    <span class="connector-conf-check text-uppercase"> <i class="fa fa-cog" aria-hidden="true"></i>
+                      Configuration
+                      <span
+                        data-ng-if="connector.configuration.length === 0 && !formHolder.connectorForm[$index].$dirty && input.selectedConfiguration.status !== 0"
+                        class="text-info">Not Configured</span>
```

Comment:
Not Configured, In progress, etc. these text can come from the consts too

Reply:
File Removed